tests/execute/escaping/main.fmf

```diff
@@ -0,0 +1,13 @@
+framework: shell
+duration: 1s
+require: []
```

Comment:
`tag: escaping` and `test: "true"` can be here as well - this will be good showcase for inheritance, only summary is different.

Reply:
It is moved to the different dir to the separate tmt test. The tag is deleted, and the test is moved to a generic statement.